examples/python-cicd-with-cloudbuilder/my_module/my_module.py

```diff
@@ -0,0 +1,72 @@
+"""
+   Copyright 2019 Google LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+Sample module used to demonstrate CI with:
+ - Pytest
+ - Cloud Source Repositories
+ - Cloud Builder
+
+Tutorial found in README.md
+
+
+
+"""
+import numpy as np
+
+
+def add(a, b):
```

Comment:
Change your method docstrings to match the styleguide: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods .

Reply:
Done. (Thanks for the education)

Reply:
You should still capitalize the start of sentences in Args, for example "a: A numeric variable." But this is getting to be major nit at this point, I'm okay with letting this okay.

Reply:
Where's that eyeroll emoji. :) 

Done.